api/apps/services/dataset_api_service.py

```
@@ -3861,20 +3938,608 @@ async def _search_layers_nav_clusters(tenant_id, dataset_id, query, top_k, embd_
+    for name, entries, _ in normalized:
+        for doc_id, value in entries:
+            if not doc_id:
+                continue
+            info = merged.get(doc_id)
+            if info is None:
+                merged[doc_id] = {"fused": value, "best": value, "legs": {name}}
+                continue
+            info["fused"] += value
+            info["legs"].add(name)
+            if value > info["best"]:
+                info["best"] = value
```

Ruff (0.16.3)

[warning] Replace `if` statement with `info["best"] = max(info["best"], value)`

Replace with `info["best"] = max(info["best"], value)`

(PLR1730)